sdk/core/Azure.Core/tests/HttpPipelineFunctionalTests.cs

```diff
@@ -267,6 +269,47 @@ public async Task TimeoutsUnbufferedBodyReads()
              testDoneTcs.Cancel();
         }
 
+        [Test]
+        public async Task SendMultipartformData()
+        {
+            IFormCollection formCollection = null;
+
+            HttpPipeline httpPipeline = HttpPipelineBuilder.Build(new TestOptions());
+            using TestServer testServer = new TestServer(
+                context =>
+                {
+                    formCollection = context.Request.Form;
+                    return Task.CompletedTask;
+                });
+
+            using Request request = httpPipeline.CreateRequest();
+            request.Uri.Reset(testServer.Address);
+
+            request.Headers.Add("Content-Type", "multipart/form-data;boundary=\"test_boundary\"");
```

Comment:
Let's add the method we've talked about so you can do.

`content.ApplyToRequest(request)`

Reply:
[Here](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Net.Http/src/System/Net/Http/MultipartContent.cs#L34) if user don't pass the boundary, [it sets the subtype as `mixed`](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Net.Http/src/System/Net/Http/MultipartContent.cs#L57), otherwise it sets `form-data`. Do we also want to support `mixed` type?